src/routes/event-match.tsx

```diff
@@ -163,12 +163,11 @@ const EventMatch = ({ eventKey, matchKey }: Props) => {
   return (
     // page setup
     <Page
-      back={`/events/${eventKey}`}
       name={
-        m.group +
-        (m.num ? ' Match ' + m.num : '') +
-        ' - ' +
-        (event ? event.name : eventKey)
+        <>
+          {m.group + (m.num ? ' Match ' + m.num : '') + ' - '}
```

Comment:
Can you use a template string (string with backticks) instead of `+`? Not sure why I used `+` when I wrote this, silly me :stuck_out_tongue_winking_eye: